common/network-types/Cargo.toml

```diff
@@ -1,33 +1,51 @@
 [package]
 name = "hopr-network-types"
-version = "0.1.0"
+version = "0.2.0"
 authors = ["HOPR Association <tech@hoprnet.org>"]
-description = "Contains types used for networking over HOPR protocol"
+description = "Contains types used for networking over the HOPR protocol"
 edition = "2021"
 homepage = "https://hoprnet.org/"
 repository = "https://github.com/hoprnet/hoprnet"
 license = "GPL-3.0-only"
 
+[features]
+default = ["runtime-async-std"]
```

Comment:
Pls consider dropping the default. Ideally the calling library will specify the requested runtime. If not, the build should fail.